src/properties.js

```diff
@@ -0,0 +1,137 @@
+
+var target = {
+    tidepool: 'tidepool',
+    diy: 'diy',
+};
+
+var unit = {
+    mmoll: 'mmol/L',
+    mgdl: 'mg/dL',
+};
+
+var screenName = {
+    settings: 'settings',
+    home: 'home',
+    bolus: 'bolus',
+    carbEntry: 'carbEntry',
+};
+
+/**
+ * @summary insulin activity model
+ * @example InsulinModel.Fiasp
+ */
+var settingInsulinModel = {
+    Walsh: 'Walsh',
+    RapidAdults: 'Rapid-Acting – Adults',
+    RapidChildren: 'Rapid-Acting – Children',
+    Fiasp: 'Fiasp'
+};
+
+/**
+ * @summary CGMModel that can be applied to the simulator
+ */
+var settingCGMModel = {
+    Constant: 'Constant',
+    SineCurve: 'Sine Curve',
+    None: 'No Data'
+};
+
+/**
+ * @summary Defaults that can be used to apply to all settings
+ * @example await settings.Apply(SettingDefault)
+ */
+var settingDefault = {
+    /**
+     * @summary DeliveryLimits: { maxBolus: '10.0', maxBasalRate: '3.0' }
```

Comment:
Are you expected to have generated documentation? If not, don't bother with these comments as they are redundant and will likely just go stale over time.